skbio/diversity/beta/_unifrac.py

```diff
@@ -0,0 +1,170 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+from skbio.util._decorator import experimental
+
+
+@experimental(as_of="0.4.0-dev")
+def unweighted_unifrac(u_counts, v_counts, otu_ids, tree):
+    """ Compute unweighted unifrac
+
+    Parameters
+    ----------
+    u_counts, v_counts: list, np.array
+        Vectors of counts of OTUs for two samples. Must be equal length.
+    otu_ids: list, np.array
+        Vector of OTU ids corresponding to tip names in ``tree``. Must be the
+        same length as ``u_counts`` and ``v_counts``.
+    tree: skbio.TreeNode
+        Tree relating the OTUs in otu_ids. The set of tip names in the tree can
+        be a superset of ``otu_ids``, but not a subset.
+
+    Returns
+    -------
+    float
+        The unweighted unifrac distance between the two samples.
+
+    Raises
+    ------
+    ValueError
+        If ``u_counts``, ``v_counts``, and ``otu_ids`` are not all of equal
+        length.
+    MissingNodeError
+        If an OTU id is provided that does not correspond to a tip in the
+        tree.
+
+    See Also
+    --------
+    weighted_unifrac
+
+    Notes
+    -----
+    Unweighted unifrac was originally described in [1]_. A discussion of
+    unweighted (qualitative) versus weighted (quantitiative) diversity metrics
+    is presented in [2]_.
+
+    References
+    ----------
+    .. [1] Lozupone, C. & Knight, R. UniFrac: a new phylogenetic method for
+       comparing microbial communities. Appl. Environ. Microbiol. 71, 8228-8235
+       (2005).
+
+    .. [2] Lozupone, C. A., Hamady, M., Kelley, S. T. & Knight, R. Quantitative
```

Comment:
I agree with @wasade on this one.  I think this reference is a nice one to have because it includes some proofs on why Unifrac is an actual distance metric.